### DISCUSSION TITLE
Cannot redeclare block-scoped variable

### DISCUSSION THREAD

**Discussion by dev02iggy** [nuxt/nuxt#16144] [General]
I have a problem.
I used to use setup(), but now I've decided to use <script setup> to make better use of await. The problem is that it seems like all pages/files that use this way manifest as if it's just one, so I'm having the problem of redeclaring the variable.
Is there a way to fix this in vue or nuxt? Or this problem can be solved with typescript as the error seems (so far) to not stop the application.

![Captura de Tela 2022-10-26 às 11 23 48](https://user-images.githubusercontent.com/83841469/198052394-4cec66cb-96c5-41fa-8160-6deb3fa70eee.png)

**Comment by yeonjulee1005** [comment]
I have the same problem.
Is there a way to set each variable globally?
or did you find another way?

**Reply by dev02iggy** [reply]
I decided to go back to the one before, because I didn't find a solution.
soon i will try again
